src/List.js

```diff
@@ -47,6 +47,25 @@ const List = () => {
     setfilteredItems(filterItems);
   };
 
+  const deleteItem = e => {
+    const deleteConfirm = window.confirm(
+      'Are you sure you want to delete this item?',
+    );
+    if (deleteConfirm === true) {
+      const db = firebase
+        .firestore()
+        .collection('items')
+        .doc(e.target.id)
+        .delete()
+        .then(function() {
+          console.log('Document successfully deleted!');
+        })
+        .catch(function(error) {
+          console.error('Error removing document: ', error);
+        });
+    }
+  };
+
```

Comment:
y'all make it look so easy! I like how you only hit the database after the user confirms on the alert window.